djstripe/models/billing.py

```diff
@@ -1462,6 +1335,14 @@ class Subscription(StripeModel):
         "because the customer was switched to a subscription to a new plan), "
         "the date the subscription ended.",
     )
+
+    # XXX: It clashes with subscription items reverse name
+    # items = JSONField(
+    #     help_text="A list of up to 20 subscription items, each with an "
+    #     "attached price.",
+    #     default="",
+    # )
+
```

Comment:
I don't think you want to store that as JSON anyway. The m2m accessor is doing its job.

Reply:
Left as TODO